ic-agent/Cargo.toml

```diff
@@ -16,7 +16,9 @@ include = ["src", "Cargo.toml", "../LICENSE", "README.md"]
 
 [dependencies]
 backoff = "0.4.0"
+cached = { version = "0.46", features = ["ahash"], default-features = false }
 candid = { workspace = true }
+ed25519-dalek = { version = "2" }
```

Comment:
Is there a reason not to use the [ed25519 library used in the ICs crypto component](https://github.com/dfinity/ic/blob/master/rs/crypto/internal/crypto_lib/basic_sig/ed25519/src/api.rs#L162)?

Reply:
None of the internal crypto crates are published to crates.io.

Reply:
The crypto team uses this library underneath: https://crates.io/crates/ed25519-consensus
According to Robin Kunzler, they had a very good reason to do so. Could you check with them if the library you intend to use is a good replacement?